### DISCUSSION TITLE
Algolia search is working but does not redirect with base url

### DISCUSSION THREAD

**Discussion by shikon** [mmistakes/minimal-mistakes#3199] [Q&A]
I got Algolia working kind of but whenever I click one of my search results it redirects to:

https://subpath

instead of https://basehostname/subpath

I have no clue what is going wrong here, anyone can help me with this?
Feel free to try at
https://shiko.nl

**Comment by iBug** [comment]
Without code it's hard to tell. Consider sharing us a public repository so we can take a look.

**Reply by shikon** [reply]
What code?
All I did was replace the algolia configuration of the sample _config.yml
What file should I share the _config.yml?

**Reply by mmistakes** [reply]
We need a public repo with your Jekyll source code to reproduce and/or suggest what might be the issue.
Looks like you're [pushing up locally built files](https://github.com/shikon/shikon.github.io)... and from glancing quickly at the HTML I don't see the correct base paths.

Since Algolia uses this HTML to generate the search index, if it's wrong there it's going to be wrong in Aloglia too.
Perhaps you built using `jekyll serve` or didn't use `JEKYLL_ENV=production` or didn't set `baseurl` properly or about a million other things. It's all speculation without seeing the source and your build scripts.

**Comment by mmistakes** [comment]
Also how you run Algolia matters too. If you're doing that all locally it could be picking up URLs without the domain which explains why they're not showing in the search results.

For example, on MM demo site I'm using Travis CI to build the Algolia search index with these configs

https://github.com/mmistakes/minimal-mistakes/blob/master/.travis.yml

**Comment by shikon** [comment]
Okay my baseurl is empty in my configuration, as the comment said it should be the subpath which is just a / .
Indeed if I check my algolia logs from their website it doesn't pick up the URLs.
https://i.imgur.com/ccYydEX.png

This might be lazy to ask, but how could get it to add my hostname?

So here is my configuration: https://github.com/shikon/shikon.github.io/blob/source/_config.yml
all the source files are available. Also if this is a security risk please let me know, so far I can not think of any.

How I build my website is by invoking a pre-push script which runs 
bundle exec jekyll build

and in my gemfile i have the jekyll algolia plugin.

**Reply by mmistakes** [reply]
Not sure. Seems like you're doing everything correct.
Probably need to check through jekyll-algolia's issue tracker as it doesn't appear to be theme related.

https://github.com/algolia/jekyll-algolia
